src/managers/tidalManager.ts

```diff
@@ -9,6 +9,7 @@ import { trayManager } from "../";
 export default class TidalManager {
 	private api: TidalAPI;
 	private currentSong: Song;
+	private lastSongTitle: string;

```

Comment:
Why do we need lastSongTitle property when songTitle is already held inside Song class which should hold data for what you want to achieve

Reply:
I made it log the song every time regardless if song title changes or not. It's done by design. There's no need to change this behavior

Reply:
The lastSongTitle property was only used in logging song data once per track. I will be reverting this change and removing the property 👍